src/bokeh/io/showing.py

```diff
@@ -55,7 +56,7 @@
 #-----------------------------------------------------------------------------
 
 def show(
-    obj: UIElement | Sequence[UIElement] | Application | ModifyDoc,
+    obj: UIElement | Sequence[UIElement] | DOMNode | Application | ModifyDoc,
```

Comment:
You may want to apply same changes to `save()` and perhaps other APIs. This then should be reflected in bokehjs as well. Besides this, perhaps `Sequence[DOMNode]` type should be allowed as well.